static/css/landing_pg.css

```diff
@@ -0,0 +1,57 @@
+ .mui--text-title{
```

Comment:
The indentation of this file doesn't really make sense to me, normally CSS looks like this:

```css
.selector {
  attribute: value;
}

.another-selector {
  …
}
```

There's probably a tool or vscode plugin that could format this automatically.

Reply:
89a10fe7b7ab17d874eaa92a9d36096460628566